src/bin/wasmer.rs

```diff
@@ -132,6 +132,10 @@ struct Run {
     )]
     backend: Backend,
 
+    /// Invoke a specified function
+    #[structopt(long = "invoke", short = "i")]
+    invoke: Option<String>,
```

Comment:
⛳️

Consider: 

```Rust
#[structopt(long = "invoke", short = "i", default = "main")]
invoke: String
``` 

This is less flexible, but for the code as it is now, this is a cleaner solution

Reply:
I don't like that much, because depending on the integration you use (emscripten, wasi, ...) the default might vary. Will go through the other approach